src/ruby.ts

```diff
@@ -248,7 +242,9 @@ export class Ruby {
   }
 
   private readRubyVersion() {
-    let dir = this.workingFolder;
+    let dir = this.customBundleGemfile
+      ? path.dirname(this.customBundleGemfile)
+      : this.workingFolder;
```

Comment:
I am confused about this one. Why can't this be just:
```suggestion
    let dir = this.cwd;
```